src/main/java/org/employeeBook/Second/DepartmentServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package org.employeeBook.Second;
+
+import org.employeeBook.Employee;
+import org.employeeBook.EmployeeService;
+import org.employeeBook.EmployeeServiceImpl;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+public class DepartmentServiceImpl extends EmployeeServiceImpl {
+
+    EmployeeServiceImpl employeeServiceImpl;
+
+    public DepartmentServiceImpl(EmployeeServiceImpl employeeServiceImpl) {
```

Comment:
```suggestion
    public DepartmentServiceImpl(EmployeeService employeeServiceImpl) {
```
инъекцию надо делать по интерфейсу, а не реализации